media-plugins/vdr-ddci2/vdr-ddci2-1.0.4.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit vdr-plugin-2
+
+if [[ "${PV}" == "9999" ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/jasmin-j/vdr-plugin-${VDRPLUGIN}.git"
+	KEYWORDS=""
+	S="${WORKDIR}/${P}"
+else
+	SRC_URI="https://github.com/jasmin-j/vdr-plugin-${VDRPLUGIN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+	S="${WORKDIR}/vdr-plugin-${VDRPLUGIN}-${PV}"
+fi
+
+DESCRIPTION="VDR plugin: DDCI2 - Digital Devices CI support"
+HOMEPAGE="https://github.com/jasmin-j/vdr-plugin-${VDRPLUGIN}"
```

Comment:
Thanks for the explanation! Went the forcepush-route and also fixed the copyright year which repoman didn't complain about before...